drivers/tty/serdev/pibridge.c

```diff
@@ -282,11 +282,14 @@ int pibridge_req_gate_tmt(u8 dst, u16 cmd, u8 *snd_buf, u8 snd_len,
 				to_receive);
 			return -EIO;
 		}
+		crc = pibridge_crc8(crc, rcv_buf, pkthdr.len);
```

Comment:
Eeeeek... buffer overflow possible here because `rcv_buf` may be smaller than `pkthdr.len`. Replace `pkthdr.len` with `to_receive` please.

Reply:
Oops thats indeed wrong. This issue slipped in due to the latest changes. It is done right for pibridge_req_io() but for pibridge_req_gate_tmt() I obviously forgot to adjust the length. Will fix it, thx